kfs-purap/src/main/java/edu/arizona/kfs/module/purap/document/PurchaseOrderRetransmitDocument.java

```diff
@@ -0,0 +1,204 @@
+/*
+ * The Kuali Financial System, a comprehensive financial management system for higher education.
+ * 
+ * Copyright 2005-2014 The Kuali Foundation
+ * 
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ * 
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ * 
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package edu.arizona.kfs.module.purap.document;
+
+import java.math.BigDecimal;
+import java.util.List;
+import org.kuali.kfs.sys.businessobject.Building;
+import edu.arizona.kfs.sys.businessobject.BuildingExtension;
+import org.kuali.kfs.module.purap.PurapConstants;
+import org.kuali.kfs.module.purap.PurapConstants.PurchaseOrderStatuses;
+import org.kuali.kfs.module.purap.businessobject.PurchaseOrderItem;
+import org.kuali.kfs.module.purap.document.service.PurchaseOrderService;
+import org.kuali.kfs.sys.context.SpringContext;
+import org.kuali.rice.core.api.datetime.DateTimeService;
+import org.kuali.rice.core.api.util.type.KualiDecimal;
+import org.kuali.rice.kew.api.exception.WorkflowException;
+import org.kuali.rice.kew.framework.postprocessor.DocumentRouteStatusChange;
+import org.kuali.rice.krad.rules.rule.event.KualiDocumentEvent;
+import org.kuali.rice.krad.workflow.service.WorkflowDocumentService;
+
+/**
+ * Purchase Order Retransmit Document
+ */
+public class PurchaseOrderRetransmitDocument extends PurchaseOrderDocument {
+    protected static org.apache.log4j.Logger LOG = org.apache.log4j.Logger.getLogger(PurchaseOrderRetransmitDocument.class);
+
+    protected boolean shouldDisplayRetransmitTab;
```

Comment:
If you extend the correct parent class, this shouldDisplayRetransmitTab property is not needed (will be inherited).